README.md

```diff
@@ -40,7 +31,28 @@ Click a folder from the list above to find out more about a particular demo, or
 | **Heap snapshot visualizer** | Source code for the [extension](https://microsoftedge.microsoft.com/addons/detail/heap-snapshot-visualizer/fceldlhognbemkgfacnffkdanocidgce). | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/heap-snapshot-visualizer) | N/A|
 | **JSON Dummy Data** | Different JSON dummy data files to use with the native JSON viewer in Microsoft Edge 110+. | N/A | [Demo](https://microsoftedge.github.io/Demos/json-dummy-data/) |
 
-#### Web Platform
+
+#### Microsoft Edge extensions
+
+| Demo | Description | Code | Live URL |
+| ---- | ----------- | ---- | -------- |
+| **Create a DevTools extension** | Learn to build a Microsoft Edge extension that extends DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-extension) | N/A |
+
+
+#### Progressive Web Apps (PWAs)
+
+| Demo | Description | Code | Live URL |
+| ---- | ----------- | ---- | -------- |
+| **1DIV** | A CSS sandbox PWA that demonstrates the Window Controls Overlay feature. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/1DIV) | [Demo](https://microsoftedge.github.io/Demos/1DIV/dist/) |
+| **Email client** | A simulated email client PWA that demonstrates how to use PWA protocol handlers. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/email-client) | [Demo](https://microsoftedge.github.io/Demos/email-client/) |
+| **PWAmp** | A music player PWA demo to read your local music files. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/pwamp) | [Demo](https://microsoftedge.github.io/Demos/pwamp/) |
+| **wami** | An image manipulation demo app to crop, resize, or add effects to images. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/wami) | [Demo](https://microsoftedge.github.io/Demos/wami/) |
+| **Temperature converter** | A simple PWA demo app that converts temperatures. Used in this tutorial: [Get started with Progressive Web Apps](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/). | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-getting-started) | [Demo](https://microsoftedge.github.io/Demos/pwa-getting-started/) |
+
+
+#### Cross-browser API samples
+
+These demos correspond to the articles in [Development tips for Microsoft Edge](https://learn.microsoft.com/en-us/microsoft-edge/web-platform/web-platform).
```

Comment:
No they don't. Not all demos on this repository have corresponding pages on LMC. This demo's purpose is larger than LMC. There may be times when we publish a blog post or a video on a topic that doesn't need to be documented on LMC, but that still requires a demo webpage.

The demos in this section in particular have no corresponding docs on LMC, and that's OK. These are demos of web platform APIs that are documented (or in the process of being documented) on MDN, mostly.

Reply:
deleted the line